src/admin/BulkUserCreationPage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from 'react';
+import { Box, Label, Heading, Container, Button, Text, Textarea, Alert, Select } from 'theme-ui';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+import { Role, CreateUserCommand } from '../api';
+import useBulkUserCreation from './hooks/useBulkUserCreation';
+
+const AnyBox = Box as any;
+
+const ROLES: Role[] = ['TRUSTED_PATIENT', 'CLINICIAN'];
```

Comment:
What should the role names be?

Reply:
```
GET /api/v1/roles
```
 😄

Reply:
What I mean is that they should come from the API and not get hard-coded here. We haven't defined them yet on the backend.

Reply:
✅ The roles are now fetched from the API.